vocabularies/UI.xml

```diff
@@ -1448,6 +1448,11 @@ or `204 No Content` if the request included a `Prefer` header with a value of `r
         </Annotation>
       </Term>
 
+      <Term Name="AdaptationHidden" Type="Core.Tag" Nullable="false" AppliesTo="Property EntitySet EntityType">
```

Comment:
```suggestion
      <Term Name="HiddenAdaptation" Type="Core.Tag" Nullable="false" AppliesTo="Property Parameter EntityType">
```
* Term name in analogy with `HiddenFilter`.
* What shall be the difference between `AppliesTo="EntitySet"` and `AppliesTo="EntityType"`? If we want container-bound annotation targets, we also need `AppliesTo="Collection"`.
* Applies also to parameters.

Reply:
`HiddenFilter` marks properties that actually used in a "hidden" filter.

The name "HiddenAdaptation" would imply that there actually is a "hidden" adaptation, whereas the term description implies that the annotated thingy is not available for adaptation.

Maybe "NoAdaptation"?

Reply:
I'm fine with "AdaptationHidden" because it is in line with "CreateHidden" etc.

Reply:
OK with `AdaptationHidden`.